helm/nessie/values.yaml

```diff
@@ -97,19 +97,26 @@ authorization:
     # allowViewingBranch: op=='VIEW_REFERENCE' && role.startsWith('test_user') && ref.startsWith('allowedBranch')
     # allowCommits: op=='COMMIT_CHANGE_AGAINST_REFERENCE' && role.startsWith('test_user') && ref.startsWith('allowedBranch')
 
-jaegerTracing:
-  # -- Specifies whether jaeger tracing for the nessie server should be enabled.
+tracing:
+  # -- Specifies whether tracing for the nessie server should be enabled.
   enabled: false
-  # -- The traces endpoint, in case the client should connect directly to the Collector, e.g. http://jaeger-collector:14268/api/traces
-  endpoint: ""
-  # -- The Jaeger service name.
-  serviceName: nessie
-  # -- Whether metrics are published if tracing is enabled.
-  publishMetrics: true
-  # -- The sampler type (const, probabilistic, ratelimiting or remote).
-  samplerType: ratelimiting
-  # -- The request sampling probability. 1=Sample all requests. Set samplerParam to somewhere between 0 and 1, e.g. 0.50, if you do not wish to sample all requests.
-  samplerParam: 1
+  # -- The collector endpoint URL to connect to (required).
+  # The endpoint URL must have either the http:// or the https:// scheme.
+  # The collector must talk the OpenTelemetry protocol (OTLP) and the port must be its gRPC port (by default 4317).
+  endpoint: "http://otlp-collector:4317"
+  # -- The sampler to use for tracing. Valid values are: off (never sample); on (always sample);
```

Comment:
Should we change it to true/false as suggested by the linter, given that the boolean's converted to `"on"` + `"off"` above?

Reply:
I think you are right. The other option is to force users to always quote "on" and "off":

Option 1: `true`, `false`, `"ratio"`
Option 2: `"on"`, `"off"`, `"ratio"`

None of these are ideal, but I think option 1 is probably easier for users.

There is actually an option 3:

Option 3: `true`, `false`, `<double>`

In this case, if  `tracing.sampler` has a double value, it resolves to `tracing.sampler=ratio` internally + `tracing.ratio=<...>`.

Wdyt? I think I can live with `true`, `false`, `"ratio"`.

Reply:
How about `sample: "all"` | `"none"` | `"0.42"`?

Reply:
Either 2 or 3 are okay - not sure, if `"ratio"` and `<double>` both work.
(We need tsyaml :disappear:)

Reply:
OK I will give @dimas-b suggestion a try, it does require some more parsing but should be doable.

Fun fact: obviously, this on/off thing hit some people before us:

https://github.com/quarkusio/quarkus/issues/30780#issuecomment-1416135542

:-)

Reply:
Done, it's much better now, thanks @dimas-b !